src/main/java/org/jabref/gui/importer/ImportEntriesViewModel.java

```diff
@@ -52,6 +55,7 @@
     private final PreferencesService preferences;
     private final BibEntryTypesManager entryTypesManager;
 
+    private final ImportCleanup cleanup = new ImportCleanup(BibDatabaseMode.BIBTEX);
```

Comment:
If one uses the user database, then this would immediately take care of https://github.com/JabRef/jabref/issues/1018 as well, right?